pair-programming/week-5/react-couter-app/README.md

```diff
@@ -1,5 +1,14 @@
-### React counter app
+### React Counter App
 
-Using codeSandbox (make sure you attach the link in the finished work column) create a counter app using react. You will use useState() to update the counter button and onClick() in each button to handle the logic of incrementing and decrementing. If you get lost you can watch [Learn useState In 15 Minutes](https://www.youtube.com/watch?v=O6P86uwfdR0) - React Hooks Explained
+You will create a counter app using React. The app should use `useState()` to update the counter and `onClick()` in each button to handle the logic for incrementing and decrementing. If you need help, you can watch [Learn useState in 15 Minutes](https://www.youtube.com/watch?v=O6P86uwfdR0) - React Hooks Explained.
+
+**Submission Options**:
+You are no longer limited to using CodeSandbox. You can submit your work using any of the following platforms:
```

Comment:
You can remove the `You are no longer limited to using CodeSandbox.` statement as the adjustment will not provide historical context of the previous statement of only using codesandbox.